microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -827,7 +827,7 @@ If you visit a shopping domain and you are an existing Bing Rebates user, Micros
 
 The shopping service is turned on by default for all users.  To change the shopping setting in Microsoft Edge, complete the following actions.
 
-1.  Go to `edge://settings/privacy`.
+1.  Go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->
```

Comment:
this should be: **edge://settings/searchServices**

 **Settings and more** (**...**) > **Settings** > **Search and services**.

Reply:
```
To change the shopping setting in Microsoft Edge, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Search and services**, 
and then turn off the **Save time and money with Shopping in Microsoft Edge** setting.
```